prospector/tools/ruff/__init__.py

```diff
@@ -77,8 +75,12 @@ def run(self, found_files: FileFinder) -> list[Message]:
                         None,
                         line=message.get("location", {}).get("row"),
                         character=message.get("location", {}).get("column"),
+                        line_end=message.get("end_location", {}).get("row"),
+                        character_end=message.get("end_location", {}).get("column"),
                     ),
                     message_str,
+                    doc_url=message.get("url"),
+                    is_fixable=bool((message.get("fix") or {}).get("applicability") in ("safe", "unsafe")),
```

Comment:
I think this could be a bit cleaner -

`message.get("fix", {})` 

but also moving the whole thing onto a separate line just to make it clearer